### DISCUSSION TITLE
Exec "go mod tidy"

### DISCUSSION THREAD

**Discussion by zhucan** [juicedata/juicefs#692] [Q&A]
![image](https://user-images.githubusercontent.com/13997135/127256597-aae60587-d794-46f3-a4f3-d0aa2a817f9f.png)

Exec "go mod tidy",  print this. but there is no effects with make build

**Comment by xiaogaozi** [comment]
Hi @zhucan, I execute `go mod tidy` in my computer doesn't have this output. Looks like it's related to your environment.

**Comment by zhucan** [comment]
@xiaogaozi  Related with my go env?

**Comment by zhucan** [comment]
![image](https://user-images.githubusercontent.com/13997135/127423522-9c7ca22e-d3a1-4062-9ae7-022c129af05b.png)
